Automattic/wp-calypso
# A4A > Feedback: Handle save feedback

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/1246
Previous PRs:
* https://github.com/Automattic/wp-calypso/pull/96934
* https://github.com/Automattic/wp-calypso/pull/97100

## Proposed Changes

* Handle sending feedback data to the API

## Why are these changes being made?


*

## Testing Instructions



* Follow the testing instructions in https://github.com/Automattic/wp-calypso/pull/97100
* Also check the following:
  * After submitting feedback, you see a success notice.
  * The feedback is posted to Slack (see D168071-code).
  * The feedback is saved to the db (see D168071-code).

<img width="693" alt="Screenshot 2024-12-09 at 2 49 06 PM" src="https://github.com/user-attachments/assets/616170bb-061d-413a-a5d4-689fc4eaba86">


## Pre-merge Checklist
